test/conformance/api/v1/errorcondition_test.go

```diff
@@ -192,7 +192,7 @@ func TestContainerExitingMsg(t *testing.T) {
 						return true, nil
 					}
 					t.Logf("Reason: %s ; Message: %s ; Status: %s", cond.Reason, cond.Message, cond.Status)
-					return true, fmt.Errorf("The configuration %s was not marked with expected error condition (Reason=\"%s\", Message=\"%s\", Status=\"%s\"), but with (Reason=\"%s\", Message=\"%s\", Status=\"%s\")",
+					return true, fmt.Errorf("The configuration %s was not marked with expected error condition (Reason=%q, Message=%q, Status=%q), but with (Reason=%q, Message=%q, Status=%q)",
```

Comment:
wanna get the first %s too?

Reply:
if names.Config is something like "values" then it reads (almost) as a normal english word, not a name.